app/services/events/post_process_service.rb

```diff
@@ -97,16 +97,16 @@ def expire_cached_charges(subscriptions)
     def handle_pay_in_advance
       return unless billable_metric
 
-      charges.where(invoiceable: false).find_each do |charge|
-        Fees::CreatePayInAdvanceJob.perform_later(charge:, event:)
-      end
-
       # NOTE: ensure event is processable
       processable_event = billable_metric.count_agg? ||
```

Comment:
We might want to add `unique_count_agg?` here 🤔

Reply:
`unique_count_agg` needs the `event.properties[billable_metric.field_name]` to be set, that's why it's not here!!±